src/Commands/AutoFullTripleLeftSwitch.cpp

```diff
@@ -0,0 +1,63 @@
+/*----------------------------------------------------------------------------*/
+/* CopyLeft (c) 2017-2018 FIRST. All Lefts Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "AutoFullTripleLeftSwitch.h"
+#include "AutoLeftSwitch.h"
+#include "PrepSwitch.h"
+#include "SetIntake.h"
+#include "SetArmPosition.h"
+#include "Delay.h"
+#include "AutoThreeLeftFromSwitch.h"
+#include "AutoThreeLeftSwitchFirstIntake.h"
+#include "AutoThreeLeftSwitchFirstIntakeReverse.h"
+#include "AutoThreeLeftToSwitch.h"
+#include "AutoThreeLeftSwitchSecondIntake.h"
+#include "AutoThreeLeftSwitchSecondIntakeReverse.h"
+#include "AutoThreeLeftSwitchThirdIntake.h"
+#include "AutoThreeLeftSwitchThirdIntakeReverse.h"
+
+AutoFullTripleLeftSwitch::AutoFullTripleLeftSwitch() {
+  //First cube
+    AddSequential(new AutoLeftSwitch);
+    AddSequential(new PrepSwitch);
+    AddSequential(new SetIntake(-0.3));
+    AddSequential(new SetArmPosition(9500));
+    AddSequential(new Delay(0.15));
```

Comment:
`SetArmPosition(9500)` seems out of place here. Should it be after `Delay(0.15)`? (a bunch of times)

The `Delay(0.150` after `SetIntake(-0.3)` is need to ensure the cube comes out. If you reverse the intake then move the arm, then the cube will likely kind of flip out.  Seems like the order should be

    AddSequential(new SetIntake(-0.3));
    AddSequential(new Delay(0.15));
    AddSequential(new SetArmPosition(9500));

Reply:
I recall now that the delay after SetArmPosition was so that the robot stopped bouncing after moving the arm.

Reply:
No change required